noir-projects/noir-protocol-circuits/crates/types/src/meta/mod.nr

```diff
@@ -524,49 +343,79 @@ pub(crate) comptime fn derive_deserialize(s: TypeDefinition) -> Quoted {
     }
 }
 
-/// Generates `Packable` implementation for a given struct and returns the packed length.
-///
-/// Note: We are having this function separate from `derive_packable` because we use this in the note macros to get
-/// the packed length of a note as well as the `Packable` implementation. We need the length to be able to register
-/// the note in the global `NOTES` map. There the length is used to generate partial note helper functions.
-pub comptime fn derive_packable_and_get_packed_len(s: TypeDefinition) -> (Quoted, u32) {
```

Comment:
We were returning the length here because the `#[note]` macro needed when auto-deriving `Packable`.

We simplified the `#[note]` macro and we no longer derive `Packable`